app/app.go

```diff
@@ -648,6 +648,8 @@ func NewApp(
 		&cdpKeeper,
 		app.distrKeeper,
 		&hardKeeper,
+		&app.mintKeeper,
```

Comment:
The mintKeeper here is referenced before it's initialized, will have to move this after the that similar to the `master` branch changes

Reply:
good catch!